Code/Framework/AzToolsFramework/AzToolsFramework/Thumbnails/ThumbnailerComponent.cpp

```diff
@@ -26,6 +26,7 @@ namespace AzToolsFramework
             : m_missingThumbnail(new MissingThumbnail())
             , m_loadingThumbnail(new LoadingThumbnail())
             , m_currentJobsCount(0)
+            , m_maxThumbnailJobs(std::thread::hardware_concurrency())
```

Comment:
It looks like we prefer the `AZStd::thread::hardware_concurrency()` API instead that has some additional cross-platform handling.

Reply:
Changed!

Reply:
Thanks! :)